### PR TITLE
Added Ace Editor Widget to README

### PR DESCRIPTION
**Pull request by Nigel2392**
Added package related to wagtail:

* https://github.com/Nigel2392/wagtail_ace_editor

### CODE REVIEW

**Comment by lb-** [comment]
Thanks @Nigel2392 - If posible can you please open up new PRs for each item. As per our guidelines here https://github.com/springload/awesome-wagtail/blob/master/.github/CONTRIBUTING.md

Especially as most of these are in different categories. 

Additionally, I've looked at a few and they do not appear to have a solid set of test cases. We appreciate the contribution to the community and maybe there can be exceptions if the packages have high usage. Nonetheless, it will be easier to review each one on its own.

**Comment by Nigel2392** [comment]
Only wagtail ace editor is added in this commit as of now

File: README.md [hunk 1]
```diff
@@ -85,6 +85,7 @@ Awesome Wagtail [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38
 - [wagtail-generic-chooser](https://github.com/wagtail/wagtail-generic-chooser) - provides base classes for building chooser popups and form widgets for the Wagtail admin, matching the look and feel of Wagtail's built-in choosers for pages, documents, snippets and images.
 - [wagtail-multi-upload](https://github.com/spapas/wagtail-multi-upload) - allows uploading of multiple related images for a page.
 - [wagtail-color-panel](https://github.com/marteinn/wagtail-color-panel) - Introduces panels for selecting colors in Wagtail.
+- [Wagtail Ace Editor](https://github.com/Nigel2392/wagtail_ace_editor) - An IDE- like code editor right in your Wagtail admin!
```
Comment (lb-) [inline README.md]:
```suggestion
- [Wagtail Ace Editor](https://github.com/Nigel2392/wagtail_ace_editor) - An IDE-like code editor right in your Wagtail admin.
```

Minor typo and let's use a simple sentence.

Thanks though, I will patch this in and merge.

Be sure to add the Package to the Wagtail classifier on PIP if you have not already done this https://pypi.org/search/?c=Framework+%3A%3A+Wagtail+%3A%3A+6

**Review by lb- — APPROVED** [review]
Thanks

**Pull request merged**